doc/modules/mod_http_notification.md

```diff
@@ -1,6 +1,6 @@
 ## Module description
 
-This module enables forwarding certain events (messages, presence, etc.) via HTTP to external services such as push (by mobile, email or SMS), big data, or analytics services.
+This module as a backend of [mod_event_pusher] that enables forwarding certain events (messages, presence, etc.) via HTTP to external services such as push (by mobile, email or SMS), big data, or analytics services.
 
 ### How it works
```

Comment:
If it's not given in the backend list of `mod_event_pusher` in the config file (or not added as `{mod_http_notification, [...]}` directly, which will have the same effect) then its functions won't be triggered by `mod_event_pusher`'s logic.

Reply:
I understand, but this line implies that `mod_http_notification` does such a check but it is now `mod_event_pusher`'s responsibility to maintain the list of enabled backends. There isn't even a "check" anyway because the list of backends is loaded from ETS for every event.

Reply:
I'm not sure what line you're referring to.

Reply:
There's only one line that mentions some checks (except for the one about `callback_module` param). :)

"checks whether http_notification is enabled (the host param is set)"

Reply:
The comment is attached to a different line, so I was confused. :<

Reply:
I couldn't attach a comment to a line so far away from the diff. :<

Reply:
:<

Reply:
(sadparrot)